Rock/Workflow/Action/CreateConnectionRequest.cs

```diff
@@ -129,6 +131,9 @@ public override bool Execute( RockContext rockContext, WorkflowAction action, Ob
                 }
             }
 
+            // Get the Comment
+            String comment = action.GetWorklowAttributeValue(GetAttributeValue(action, "ConnectionCommentAttribute").AsGuid());
```

Comment:
@bwitting Is the plan to use this comment as the connectionRequest.Comments property?  If so I think you forgot that part, right?